pkg/cmd/target/kubernetes/view/view.go

```diff
@@ -0,0 +1,48 @@
+package view
+
+import (
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/OctopusDeploy/cli/pkg/cmd"
+	"github.com/OctopusDeploy/cli/pkg/cmd/target/shared"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/OctopusDeploy/cli/pkg/factory"
+	"github.com/OctopusDeploy/cli/pkg/machinescommon"
+	"github.com/OctopusDeploy/cli/pkg/output"
+	"github.com/OctopusDeploy/cli/pkg/usage"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/machines"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdView(f factory.Factory) *cobra.Command {
+	flags := shared.NewViewFlags()
+	cmd := &cobra.Command{
+		Args:  usage.ExactArgs(1),
+		Use:   "view {<name> | <id>}",
+		Short: "View a Kubernetes deployment target",
+		Long:  "View a Kubernetes deployment target in Octopus Deploy",
+		Example: heredoc.Docf(`
+			$ %[1]s deployment-target kubernetes view 'target-name'
+		`, constants.ExecutableName),
+		RunE: func(c *cobra.Command, args []string) error {
+			opts := shared.NewViewOptions(flags, cmd.NewDependencies(f, c), args)
+			return ViewRun(opts)
+		},
+	}
+
+	machinescommon.RegisterWebFlag(cmd, flags.WebFlags)
+
+	return cmd
+}
+
+func ViewRun(opts *shared.ViewOptions) error {
+	return shared.ViewRun(opts, contributeEndpoint, "Kubernetes")
+}
+
+func contributeEndpoint(_ *shared.ViewOptions, targetEndpoint machines.IEndpoint) ([]*output.DataRow, error) {
+	data := []*output.DataRow{}
+	endpoint := targetEndpoint.(*machines.KubernetesEndpoint)
+
+	data = append(data, output.NewDataRow("Kubernetes Cluster URL", endpoint.ClusterURL.String()))
```

Comment:
The cluster url property isn't used for all auth types
![image](https://user-images.githubusercontent.com/5336529/208020971-66b3866e-25cd-445c-9953-1625f26f95f0.png)


The view command should probably include the auth type, and perhaps some specific detail for each type